research/delf/delf/python/training/model/resnet50.py

```diff
@@ -54,23 +55,23 @@ def __init__(self, kernel_size, filters, stage, block, data_format):
     bn_axis = 1 if data_format == 'channels_first' else 3
 
     self.conv2a = layers.Conv2D(
-        filters1, (1, 1), name=conv_name_base + '2a', data_format=data_format)
+      filters1, (1, 1), name=conv_name_base + '2a', data_format=data_format)
```

Comment:
I see these padding changes in the method arguments from 4 spaces to 2 spaces from line 57 to line 298 are not consistent with our internal codebase. Is this expected?